client/web/src/insights/pages/dashboards/creation/components/insights-dashboard-creation-content/utils/get-global-subject-tooltip-text.ts

```diff
@@ -10,7 +10,7 @@ export function getGlobalSubjectTooltipText(globalSubject: SettingsSiteSubject |
 
     const globalSubjectAdminCheckMessage = globalSubject.viewerCanAdminister
         ? undefined
-        : "You don't have a permission to change global scope. Reach out your site admin"
+        : "You don't have permission to change the global scope. Reach out to your site admin"
```

Comment:
@Joelkw @AlicjaSuska @felixfbecker these tooltip texts require proofreading.

Reply:
I might just shorten/combine the sentences:
```suggestion
        : "Only site admins can change global settings"
```
Ideally I'd say "Only site admins can define global insights" or "[...] dashboards" if we know here whether this is for an insight or dashboard.